app/controllers/statistics_controller.rb

```diff
@@ -14,6 +14,11 @@ def index
       format.json do
         render json: StatisticsFilterJsonPresenter.new(@filter, view_context, PublicationesquePresenter)
       end
+      format.atom do
+        documents = Publicationesque.published_with_eager_loading(@filter.documents.map(&:id))
+        @statistics = Whitehall::Decorators::CollectionDecorator.new(
+          documents.sort_by(&:public_timestamp).reverse, PublicationesquePresenter, view_context)
```

Comment:
Minor thing, but you might want to put the closing bracket on a new line and each argument on its own line for better readability. Like this:
```
Whitehall::Decorators::CollectionDecorator.new(
  documents.sort_by(&:public_timestamp).reverse,
  PublicationesquePresenter, 
  view_context
)
```